feature/home/src/main/java/com/yapp/feature/home/component/NoticeSection.kt

```diff
@@ -0,0 +1,60 @@
+package com.yapp.feature.home.component
+
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.yapp.core.designsystem.component.button.outlined.YappOutlinedAssistiveButtonMedium
+import com.yapp.core.designsystem.component.chip.ChipColorType
+import com.yapp.core.designsystem.theme.YappTheme
+import com.yapp.core.ui.component.NoticeInfo
+import com.yapp.core.ui.component.NoticeItem
+import com.yapp.core.ui.component.TagInfo
+import com.yapp.feature.home.R
+
+@Composable
+fun NoticeSection(
+    noticeInfo: List<NoticeInfo>,
+    onMoreButtonClick: () -> Unit,
+) {
+    HomeSectionBackground {
+        Text(
+            text = stringResource(R.string.home_notice_title),
+            color = YappTheme.colorScheme.labelNormal,
+            style = YappTheme.typography.headline1Bold
+        )
+        Spacer(Modifier.height(8.dp))
+        repeat(minOf(3, noticeInfo.size)) {
+            val isLastItem = it == 2
+            NoticeItem(noticeInfo = noticeInfo[it], isLastItem = isLastItem)
+        }
```

Comment:
```suggestion
        repeat(minOf(3, noticeInfo.size)) {
            val isLastItem = it == 2
            val modifier = if (isLastItem) Modifier.borderBottom() else Modifier
            NoticeItem(
            modifier = modfiier,
            noticeInfo = noticeInfo[it], 
            )
        }
```

A: 요런 느낌입니다~! (https://github.com/YAPP-admin/yappu-world-android/pull/77/files#r1946108332)